.gitignore

```diff
@@ -0,0 +1,6 @@
+.*.swo
+.*.swp
+
+testdata/
```

Comment:
the go tool ignores testdata, please remove this line

if something is writing into testdata/ let's fix that in a followup